dace/config_schema.yml

```diff
@@ -770,6 +770,184 @@ required:
                 title: Print FPGA runtime 
                 description: Prints the runtime of instrumented FPGA kernel states to standard output. 
 
+    #############################################
+    # SOAP I/O analysis settings
+
+    soap:
+        type: dict
+        title: SOAP
+        description: SOAP configuration parameters
+        required:
+            analysis:
+                type: dict
+                title: Analysis
+                description: SOAP analysis parameters
+                required:         
+                    io_analysis:
+                        type: bool
+                        title: Perform I/O analysis
+                        default: true
+                        description: >
+                            Specifies whether the I/O analysis on the SDFG shall be performed
+
+                    wd_analysis:
+                        type: bool
+                        title: Perform work/depth analysis
+                        default: false
+                        description: >
+                            Specifies whether the work-depth analysis on the SDFG shall be performed
+
+                    all_injective:
+                        type: bool
+                        title: Projection of non-SOAP accesses
+                        default: true 
+                        description: >
+                            If access function vectors are not SOAP (e.g., A[i,k] and A[k,j]), should it
+                            be assumed that they never overlap and analyze them as separate logical arrays.
+                    
+            decomposition:
```

Comment:
Having decomposition parameters in the config doesn't hurt tbh, as long as you can change them programmatically. Is it possible to change the `chosen_par_setup` when calling the solver?